docs/simulators/OpenFOAM.md

```diff
@@ -126,6 +126,286 @@ task.download_outputs()
+
+### Running the Simulation
+
+Running the script above will take some time, as OpenFOAM tasks like
+`snappyHexMesh` and `checkMesh` can be computationally intensive, especially
+with large meshes. The `task.wait()` command will block until the simulation
+is complete, after which the output files will be available.
+
+### Post-Simulation
+
+After the simulation completes, the output files will be downloaded locally by
+calling `task.download_outputs()`. The outputs will include simulation results
+like mesh quality reports, final results, and log files for each step in the process.
+
+To inspect the output, look under the `inductiva_output` folder where the
+results are downloaded:
+
+```bash
+ls -lasgo inductiva_output/<task_id>
+total 672
+  0 drwxr-xr-x  143     4576 Sep 19 16:06 .
+  0 drwxr-xr-x  400    12800 Sep 19 16:05 ..
+  0 drwxr-xr-x   11      352 Sep 19 16:06 0.orig
+  8 -rw-r--r--    1      626 Sep 19 16:06 Allclean
+ 16 -rw-r--r--    1     6998 Sep 19 16:06 Allrun
+  8 -rw-r--r--    1      991 Sep 19 16:06 COPYING
+ 48 -rw-r--r--    1    21547 Sep 19 16:06 README.md
+  0 -rw-r--r--    1        0 Sep 19 16:06 case.foam
+  0 drwxr-xr-x    8      256 Sep 19 16:06 constant
+  0 drwxr-xr-x   27      864 Sep 19 16:06 dynamicCode
+  0 drwxr-xr-x   13      416 Sep 19 16:06 figures
+  0 drwxr-xr-x    4      128 Sep 19 16:06 processor0
+  0 drwxr-xr-x    4      128 Sep 19 16:06 processor1
+  0 drwxr-xr-x    4      128 Sep 19 16:06 processor10
+    ...
+  0 drwxr-xr-x    4      128 Sep 19 16:06 processor126
+  0 drwxr-xr-x    4      128 Sep 19 16:06 processor127
+ 32 -rw-r--r--    1    14437 Sep 19 16:06 stderr.txt
+536 -rw-r--r--    1   272331 Sep 19 16:06 stdout.txt
+  0 drwxr-xr-x   30      960 Sep 19 16:06 system
+ 24 -rw-r--r--    1    11399 Sep 19 16:06 thumbnail.png
+```
+
+You can now perform post-processing on the results, just as you would if you had
+run the simulation locally.
+
+### Utilizing an MPI Cluster for Enhanced Simulation Performance
+
+The initial simulation conducted on a single `c3d-highcpu-360` machine took 40
+hours and 20 minutes to complete. To expedite this process, we opted to deploy
+the simulation across two `c3d-highcpu-360` machines in an MPI cluster configuration.
+This strategic change reduced the simulation time significantly to just 19 hours
+and 10 minutes.
+
+Leveraging Inductiva's Python library, scaling the simulation resources is
+straightforward. You simply need to modify the instantiation of the machine group,
+replacing it with an MPI cluster setup, and adjust the `n_vcpus` parameter accordingly.
+Below is an example of the code implementation:
+
+```python notest
+auto_terminate = (datetime.datetime.now() + datetime.timedelta(days=6)).replace(tzinfo=pytz.utc)
+mpi_cluster = inductiva.resources.MPICluster(
+    machine_type="c3d-highcpu-360",
+    data_disk_gb=300,
+    num_machines=2,
+    auto_terminate_ts=auto_terminate
```

Comment:
This should not be needed. Please submit and issue to get the desired behaviour. Please don't let these things accumulate (plus, this ends in less than 24hours so it shouldn't even be an issue)

Reply:
The issue already exists https://github.com/inductiva/tasks/issues/389
I can remove this from here, in the expectation that in the near future this will be fixed (and you are correct, for this case we would not need it but i didn't know it at the time)

Reply:
Please remove it here. Let's assume it will be fixed somewhere else.